src/Authentication/Authentication/Common/DiskDataStore.cs

```diff
@@ -0,0 +1,228 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Graph.PowerShell.Authentication
+{
+    using System.IO;
+    using System.Text;
+
+    /// <summary>
+    /// Disk data store based on Windows file management APIs (System.IO).
```

Comment:
haven't read through where this gets used yet but don't we need this cross plat?

Reply:
System.IO APIs are cross platform in .NetCore. I'll update the comment.